hadoop-hdfs-project/hadoop-hdfs-rbf/pom.xml

```diff
@@ -177,6 +177,11 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>junit-jupiter-params</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
`mvn test -Dtest=TestObserverWithRouter` skips ParameterizedTest if we don't add junit-jupiter-engine here.

Reply:
can be a separate ticket

Reply:
Agreed, I deleted it by 0e66511214db118df601b41670c1446e63f97f49. I will create another ticket.